### DISCUSSION TITLE
[Piano] Deck the Halls (No JSON)

### DISCUSSION THREAD

**Discussion by Abc2557** [Monika-After-Story/MonikaModDev#7794] [general - piano notes]
Deck the halls notes on the piano:
O I U Y T Y U T Y U I Y U Y T R T
O I U Y T Y U T Y U I Y U Y T R T
Y U I Y U I O U I O P P [ ] [ P O
O I U Y T Y U T Y U I Y U Y T R T
Because I have no clue of how to make a JSON thing (whatever it is), I'm just leaving the notes here for anyone who _does_ know how to.
Good luck to whoever may decide to make the JSON thing- Happy New Year's Eve / New Year's Day!

**Comment by briaryoung** [comment]
Will add this to my list of piano songs to do, ty
